src/parser.js

```diff
@@ -112,10 +114,10 @@ const parser = yargs
         - true - Example: { foo: bar }
         - false - Example: {foo: bar}
       `,
-      choices: [true, false],
```

Comment:
I didn't make much sense when I added it in the previous commit. It's encouraged to be used as either `--semi` or `--no-semi`. Furthermore, yargs complains that `undefined` is not a valid choice (`true` or `false`).